pyproject.toml

```diff
@@ -47,13 +47,15 @@ python-dotenv = "1.0.0"
 pyinstaller = "^5.12.0"
 setuptools-lint = "^0.6.0"
 sphinx = "7.0.1"
-nicegui = "1.2.16"
+nicegui = "1.2.21"
 spacy = "^3.5.3"
 fuzzywuzzy = "0.18.0"
 segment-anything = "^1.0"
 torchvision = "^0.15.2"
 sumy = "0.11.0"
 nltk = "3.8.1"
+pystray = "0.19.4"
```

Comment:
From https://pystray.readthedocs.io/en/latest/usage.html#displaying-notifications:

> Displaying notifications
> This is not supported on macOS and Xorg; please check Icon.HAS_NOTIFICATION at runtime for support on the current platform.

Are there alternatives which do support notifications cross platform?

Reply:
That's weird, the notifications are working for me on MacOS. Maybe they didn't update the docs? The way they specify the call to notify() doesn't work since the notification needs a title. I am open to using a different library to handle notifications though, since this is just using osascript - which isn't very configurable from pystray.